.travis.yml

```diff
@@ -35,16 +37,25 @@ script:
 - kubectl get pods --all-namespaces
 - make test
 before_deploy:
-- git config --local user.name "copejon"
-- git config --local user.email "copejon@redhat.com"
-- source version && git tag -f $RELEASE_TAG
+- git remote add https://${GH_TOKEN}@github.com/kubevirt/containerized-data-importer.git
+- docker login -u="$DOCKER_USER" -p="$DOCKER_PASS"
 deploy:
+# Increment version if tagged
 - provider: script
   on:
     branch: master
     repo: kubevirt/containerized-data-importer
-  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" && make release
+    tags: true
+  script: hack/version-and-deploy.sh $TRAVIS_TAG
   skip_cleanup: true
+# Release new docker images
+- provider: script
+  script: make release
+  skip_cleanup: true
+  on:
+    branch: master
```

Comment:
it's really just another safety catch for human error.  if someone is on a non-master branch and pushes a tag, this will stop it from overwriting the values.